backend-node/src/index.ts

```diff
@@ -189,6 +190,14 @@ const route = createRoute({
 })
 const app = new OpenAPIHono()
 
+app.use(
+  "*",
+  sentry({
+    dsn: process.env.SENTRY_DSN,
```

Comment:
We should probably use a different variable name as it should also be a different DSN for the backend